content/style-injector.js

```diff
@@ -30,12 +32,28 @@ function createStyleInjector({compare, setStyleContent, onUpdate}) {
     createStyle
   };
 
+  function _initCreationDoc() {
+    // FF59+ workaround: allow the page to read our sheets, https://github.com/openstyles/stylus/issues/461
+    // Since browser version can be spoofed, checking for getPreventDefault that was removed in FF59
+    const pageContextDoc = !Event.prototype.getPreventDefault && document.wrappedJSObject;
+    creationDoc = pageContextDoc || document;
+    if (pageContextDoc) {
+      const el = createStyle('test', '');
+      const applied = el.sheet;
```

Comment:
The element has to be attached to DOM before checking `el.sheet`. BTW, we can put a comment referencing https://bugzilla.mozilla.org/show_bug.cgi?id=1579345#c3.

Reply:
Yep, my bad.